MotorPair.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "motor_class.h"
+
+class MotorPair
+{
+	motor *left, *right;
```

Comment:
It would be better to just use regular objects instead of pointers.
A pointer here doesn't gain us anything.